.gitignore

```diff
@@ -1,6 +1,8 @@
 # See http://help.github.com/ignore-files/ for more about ignoring files.
 
 .yarn-cache
+.yarnrc
+.yarn
```

Comment:
Why?

Reply:
To avoid files which are not on main but on this branch to be pushed.

Reply:
Okay but why did you add this as part of the pr?